pkg/hubble/metrics/policy/handler.go

```diff
@@ -66,3 +75,19 @@ func (d *policyHandler) ProcessFlow(ctx context.Context, flow *flowpb.Flow) erro
 	d.verdicts.WithLabelValues(labels...).Inc()
 	return nil
 }
+
+func (d *policyHandler) ProcessFlowL7(ctx context.Context, flow *flowpb.Flow) error {
+	labelValues, err := d.context.GetLabelValues(flow)
+	if err != nil {
+		return err
+	}
+
+	direction := strings.ToLower(flow.GetTrafficDirection().String())
+	match := "l7"
```

Comment:
might be useful to use L7 protocol names as the label here, like `http`, `dns` or `kafka`. what do you think?

Reply:
I like that, or maybe something like `l7/<proto>` or `l7-<proto>`